cmd/observer/node_builder/observer_builder.go

```diff
@@ -0,0 +1,839 @@
+		flags.UintVar(&builder.rpcConf.MaxHeightRange, "rpc-max-height-range", defaultConfig.rpcConf.MaxHeightRange, "maximum size for height range requests")
+		flags.StringSliceVar(&builder.rpcConf.PreferredExecutionNodeIDs, "preferred-execution-node-ids", defaultConfig.rpcConf.PreferredExecutionNodeIDs, "comma separated list of execution nodes ids to choose from when making an upstream call e.g. b4a4dbdcd443d...,fb386a6a... etc.")
+		flags.StringSliceVar(&builder.rpcConf.FixedExecutionNodeIDs, "fixed-execution-node-ids", defaultConfig.rpcConf.FixedExecutionNodeIDs, "comma separated list of execution nodes ids to choose from when making an upstream call if no matching preferred execution id is found e.g. b4a4dbdcd443d...,fb386a6a... etc.")
+		flags.BoolVar(&dummyBool, "log-tx-time-to-finalized", false, "deprecated - log transaction time to finalized")
+		flags.BoolVar(&dummyBool, "log-tx-time-to-executed", false, "deprecated - log transaction time to executed")
+		flags.BoolVar(&dummyBool, "log-tx-time-to-finalized-executed", false, "deprecated - log transaction time to finalized and executed")
+		flags.BoolVar(&dummyBool, "retry-enabled", false, "deprecated - whether to enable the retry mechanism at the access node level")
+		flags.BoolVar(&dummyBool, "rpc-metrics-enabled", false, "deprecated - whether to enable the rpc metrics")
+		flags.StringToIntVar(&builder.apiRatelimits, "api-rate-limits", defaultConfig.apiRatelimits, "per second rate limits for Access API methods e.g. Ping=300,GetTransaction=500 etc.")
+		flags.StringToIntVar(&builder.apiBurstlimits, "api-burst-limits", defaultConfig.apiBurstlimits, "burst limits for Access API methods e.g. Ping=100,GetTransaction=100 etc.")
+		flags.BoolVar(&dummyBool, "staked", false, "deprecated - whether this node is a staked access node or not")
+		flags.StringVar(&builder.observerNetworkingKeyPath, "observer-networking-key-path", defaultConfig.observerNetworkingKeyPath, "path to the networking key for observer")
+		flags.StringSliceVar(&builder.bootstrapNodeAddresses, "bootstrap-node-addresses", defaultConfig.bootstrapNodeAddresses, "the network addresses of the bootstrap access node if this is an observer e.g. access-001.mainnet.flow.org:9653,access-002.mainnet.flow.org:9653")
+		flags.StringSliceVar(&builder.bootstrapNodePublicKeys, "bootstrap-node-public-keys", defaultConfig.bootstrapNodePublicKeys, "the networking public key of the bootstrap access node if this is an observer (in the same order as the bootstrap node addresses) e.g. \"d57a5e9c5.....\",\"44ded42d....\"")
+		flags.StringVar(&dummyString, "public-network-address", "", "deprecated - access node's public network bind address")
+		flags.DurationVar(&builder.apiTimeout, "flow-api-timeout", defaultConfig.apiTimeout, "tcp timeout for Flow API gRPC socket")
+	}).ValidateFlags(func() error {
```

Comment:
You can just remove all the `dummy*` flags. No need to deprecate them since the observer is new

Reply:
Fix: https://github.com/GetElastech/flow-go/commit/0c9c32c2232f7893fb43551163d7f26916fe4386